addon/components/o-s-s/button-dropdown.ts

```diff
@@ -57,14 +57,14 @@ export default class extends Component<OSSButtonDropdownArgs> {
 
   @action
   setupChildrenClickHandler(element: HTMLElement): void {
-    element.querySelectorAll('.oss-button-dropdown__item').forEach((child: HTMLElement) => {
+    element.querySelectorAll('.oss-button-dropdown__item').forEach((child: Element) => {
```

Comment:
isn't `HTMLElement` a more precise `Element`? Aren't we losing precision with this change ?